### PR TITLE
Homework "grammar" has been done.

### PR DESCRIPTION
**Pull request by Hopd84**

### CODE REVIEW

**Review by olgaz-skiba — DISMISSED** [review]
Igfm

File: homework/english/byda_artem/lecture1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Упражнения Present Simple и Present Continuous
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum is working today?
```
Comment (Alex99905) [inline homework/english/byda_artem/lecture1/grammar.md:5]: Is your mum working today?

File: homework/english/byda_artem/lecture1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+2. Why this boys are talking now?
```
Comment (Alex99905) [inline homework/english/byda_artem/lecture1/grammar.md:6]: Why are this boys talking now?

**Review by Alex99905 — APPROVED** [review]

File: homework/english/byda_artem/lecture1/grammar.md (hunk 2 above)
Comment (Hopd84) [reply homework/english/byda_artem/lecture1/grammar.md:6]: TY for your observation!

**Review by olgaz-skiba — APPROVED** [review]

**Review by Svetlana101010 — APPROVED** [review]

**Review by olgaz-skiba — APPROVED** [review]
Igfm

**Review by Alex99905 — APPROVED** [review]

**Pull request merged**